### PR TITLE
fix(replays): Replay duration value on the Issue Details page 

### LINKED ISSUE #38659: Bug: The replay duration value is wrong on the Issue Details page

### PR DESCRIPTION
**Pull request by jesus4497**
### Changes
- Fixed the replay duration value by sending the seconds
- Added i18n to a missing text that I encountered

Closes #38659 

image

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Review by ryan953 — APPROVED** [review]

File: static/app/components/events/eventReplay/replayContent.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    );
+
+    // Expect Duration value to be correct
+    expect(screen.getByText('URL')).toBeInTheDocument();
+    expect(screen.getByTestId('replay-duration')).toHaveTextContent('11 minutes');
+
+    // Expect Timestamp value to be correct
+    expect(screen.getByText('Timestamp')).toBeInTheDocument();
+    expect(screen.getByTestId('replay-timestamp')).toHaveTextContent(
+      'Oct 17, 2017 2:41:20 AM UTC'
```
Comment (ryan953) [inline static/app/components/events/eventReplay/replayContent.spec.tsx:158]: does the mock setup this timestamp? it's a 2017 date.

Comment (jesus4497) [reply static/app/components/events/eventReplay/replayContent.spec.tsx]: It was, but I decided that it was better to make this mocked date value a bit more robust - made the change in the last commit

File: static/app/components/events/eventReplay/replayContent.spec.tsx [hunk 2]
```diff
@@ -30,8 +30,8 @@ const mockedReplay: Partial<ReplayReader> = {
     dist: '',
     traceIds: [],
     errorIds: ['5c83aaccfffb4a708ae893bad9be3a1c'],
-    startedAt: new Date(),
-    finishedAt: new Date(),
+    startedAt: new Date(1663025353247.3),
```
Comment (ryan953) [inline static/app/components/events/eventReplay/replayContent.spec.tsx:33]: is the `.3` significant? it's already millieseconds

Comment (ryan953) [inline static/app/components/events/eventReplay/replayContent.spec.tsx]: lets do something that's easy to read like:

```
const mockStartedAt = 'Sep 12, 2022 11:29:13 PM UTC';
const mockFinishedAt = 'Sep 15, 2022 17:22:07 PM UTC';

...
startedAt: new Date(mockStartedAt),
finishedAt: new Date(mockFinishedAt)
...

expect(...)..toHaveTextContent(mockStartedAt);
```

Then i don't have to manually convert milliseconds to know what the test is doing.

**Review by ryan953 — APPROVED** [review]

**Pull request merged**